src/actions/hydration.js

```diff
@@ -11,5 +11,4 @@ export const hydrate = amount => ({
   amount,
 });
 
-export const drinkWater = () => hydrate(10);
-export const drinkWine = () => hydrate(3);
+export const drink = liquid => hydrate(liquid.hydration);
```

Comment:
Start of the higher level/data-driven idea here. Rather than hard-coding your fluids into your action, you defer the amount of hydration to the think being drunk.